README.md

```diff
@@ -1,8 +1,42 @@
-# Examples for MALOS-eye
+# MALOS-eye examples
 
-* Demographics example. Detect faces and get age, gender, emotion and pose by querying a remote server.
+### Pre-Requisites
+```
+# Base repository and assoc. packages
+echo "deb http://packages.matrix.one/matrix-creator/ ./" | sudo tee --append /etc/apt/sources.list;
+sudo apt-get update;
+sudo apt-get upgrade;
+sudo apt-get install libzmq3-dev xc3sprog matrix-creator-openocd wiringpi cmake g++ git;
```

Comment:
Missing matrix-creator-init, malos-eye and matrix-creator-malos from the package list.